### ISSUE TITLE
Missing sensor for  "Additional Production - today"

### ISSUE THREAD

**Issue by Peyto79** [torbennehmer/hacs-e3dc#145]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/torbennehmer/hacs-e3dc/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

No, I dont think so.

### Describe the solution you'd like

There are several "today" sensors but the one for the additional source is missing. 
"Solar Production - today" does only show the daily sum of the E3DC production and I want to sum it with my additional external production.

### Describe alternatives you've considered

None.

**Comment by Thomas-sw11** [comment]
Yes, same for me. Additional today would be nice to have as well.

**Comment by schabert** [comment]
+1

**Comment by macrauder** [comment]
+1

**Comment by uncle03** [comment]
+1

**Comment by macrauder** [comment]
+1

**Comment by ulkrueger** [comment]
+1

**Comment by tammof** [comment]
+1

**Comment by fummelfichte** [comment]
+1

I think a combination of helpers (integral followed by utility meter) based on the sensor `S10E_Compact Additional` could be a work-around

**Comment by chrispi68** [comment]
+1

**Comment by MacSchierer** [comment]
+1
